reflex/app.py

```diff
@@ -759,9 +827,7 @@ async def on_event(self, sid, data):
         # Process the events.
         async for update in process(self.app, event, sid, headers, client_ip):
             # Emit the event.
-            await asyncio.create_task(
-                self.emit(str(constants.SocketEvent.EVENT), update.json(), to=sid)
-            )
+            await self.emit(str(constants.SocketEvent.EVENT), update.json(), to=sid)
```

Comment:
Good cleanup, not sure why we needed that originally but all the event chain stuff seems to be working well in my testing